wait-for-idle-runner (new action) (#170)
Homebrew/actions: wait-for-idle-runner/Readme.md

# Wait for idle runner GitHub Action

An action that waits for an idle runner.

> The `result` output is set to reflect the state of the runner.

Reviewer: Sorry to iterate more on this but I think something like this makes a little more sense now that `runner-found` and `runner-idle` are being used:

```suggestion
The `runner-found` output is set to `true` or `false` based on whether or not the runner was found.
If the runner was found, the `runner-idle` output is set to `true` or `false` based on whether the runner is idle.
```

> It returns "::set-output name=result::true" when the runner is idle.

Reviewer: I'd maybe say something like this instead, as it doesn't really _return_ that text

```suggestion
The `result` output is set to reflect the state of the runner.
```

Feel free to change that to better match what the `result` output really is.

# Wait for idle runner GitHub Action

An action that waits for an idle runner.
It returns "::set-output name=result::true" when the runner is idle.

## Usage

```yaml
- name: Wait for idle runner
  uses: Homebrew/actions/wait-for-idle-runner@master
  with:
    github_token: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
    runner_name: linux-self-hosted-1
    repo_name: ${{ github.event.repository.full_name }}
```

> 

Reviewer 2: ```suggestion
```
